clients/src/main/java/org/apache/kafka/common/serialization/Serdes.java

```diff
@@ -48,6 +59,11 @@
 
     static public final class DoubleSerde implements Serde<Double> {
         @Override
+        public void configure(Map<String, ?> configs, boolean isKey) {
+            // do nothing
```

Comment:
I think I would drop all of these `// do nothing` comments. Do we need them for checkstyle?